source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,195 @@
+
+**Driver Station Display and Log**
+
+When the robot is plugged in and connected to the driver station laptop, the battery voltage is displayed on the NI Driver Station software.
+
+After you finish a driving session, you can :ref:`review the battery voltage in the Log Viewer. <docs/software/driverstation/driver-station-log-viewer:Using the Graph Display>`
+
+**Voltmeter** or **Multimeter**
+
+A voltage reading directly at the battery will give you a snapshot of what the Voc (Voltage open circuit, or "float voltage") is in the "Unloaded" state. Anything from 13.0 to 14.0 may show up for charged batteries, but depends as much on the individual battery as it does on real state of charge. Below 13V, it's worth trying to charge it more, but in general this is not a recommended method for understanding battery health: the open circuit voltage is not as useful as the combination of internal resistance and voltages at specific loads provided by a Load Tester (or Battery Analyzer).
+
+**Load Tester**
+
+A battery load tester can be used as a quick way to determine the detailed readiness of a battery. It may provide information like: open-load voltage, voltage under load, internal resistance, and state of charge. These metrics can be used to quickly confirm that a battery is ready for a match and even help to identify some long term problems with the battery.
+
+.. image:: images/robot-battery/beak.png
+  :alt: Output screen of a common load tester showing properties of the battery.
+
+Ideal internal resistance should be less than 0.015 Ohms. The manufacturer specification for most batteries is 0.011 Ohms. If a battery gets higher than 0.020 Ohms it is a good idea to consider not using that battery for competition matches.
+
+If a battery shows significantly lower voltages at the higher test current loads, it may not be done charging, or it may need to be retired.
+
+Understanding Battery Voltages
+------------------------------
+
+A "12V battery" is anything but 12V.
+
+Fully charged, a battery can be anywhere from 13 to 14 volts open circuit (Voc). Open circuit voltage is measured with *nothing* connected.
+
+Once a load (like a robot) is connected, and any amount of current is flowing, the battery voltage will drop. So if you check a battery with a Voltmeter, and it reads 13.5, and then connect it to your robot and power on, it will read lower, maybe 13.1 on the Driver Station display. Those numbers will vary with every battery and specific robot, see Characterization below. Once your robot starts running, it will pull more current, and the voltage will drop further.
+
+Depleted batteries will read between 11.7 and 12.5 volts on an idle robot. Try to swap the batteries before they start reaching brownout safety thresholds, as frequently entering low voltage ranges risks permanent battery damage.
```

Comment:
https://www.researchgate.net/publication/315847359_Modeling_and_Validation_of_12V_Lead-Acid_Battery_for_Stop-Start_Technology

It might be 80% on a flooded car battery, but doesn't indicate much over 50% on sealed gel/agm. 

Anecdotally, our robots are difficult to drive below 12.5V - but I also encourage aggressive gearing in our custom drives, and bone stock kitbot should be fine down to 12. 

This is why Voc is not recommended for understanding battery health ;-)

Reply:
I will die on the hill that 12.5v is an acceptable upper replacement threshold ;-p

Reply:
I agree that 12.5 is a good number.  I think the important thing here is the voltage that teams should be checking for before a match and I think we should make it very clear (we do) that starting with less that 12.5 is just a bad idea in most cases.

Reply:
That chart you linked shows 12.5V at ~70% SoC. That is certainly lower than the generic red-yellow-green charts you get when googling and does seem to match up closer with some manufacturer charts. 50% SoC is a better spot for battery life than what I originally suggested, perhaps bumping down to 12.3 per the paper you cited?

Reply:
Maybe instead of "Depleted batteries will read between 11.7 and 12.5 volts on an idle robot." we could say:
"Batteries reading less than 12.5V on an idle robot should not be used for a match"

Reply:
I'm fine with either 12.3 or the direction Jason proposed.

Reply:
Updated to match Jason's wording